dev-lang/tk/tk_x-8.6.12.recipe

```diff
@@ -0,0 +1,103 @@
+SUMMARY="A windowing toolkit for use with Tcl"
```

Comment:
I would omit this recipe as it isn't ready for use yet. See: https://github.com/waddlesplash/xlibe/issues/2